samples/subsys/logging/kernel_event_logger/README.rst

```diff
@@ -0,0 +1,40 @@
+.. _kerneleventlogger_sample:
+
+Kerneleventlogger Sample
+######################
+
+Overview
+********
+
+A simple application that demonstrates use of kernel event
+logger feature.Two threads (A and B) of same priority are
```

Comment:
add a space after the period ending a sentence

Reply:
k